stdlib/public/Compatibility50/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(swift_runtime_compile_flags ${SWIFT_RUNTIME_CORE_CXX_FLAGS})
+set(swift_runtime_linker_flags ${SWIFT_RUNTIME_CORE_LINK_FLAGS})
+
+if(SWIFT_BUILD_SDK_OVERLAY)
```

Comment:
@gottesmm What's the best way to set up the CMake file so this only builds for Darwin targets?

Reply:
Provide an explicit list of target SDKs, like the Platform overlay does.

Reply:
@compnerd

Reply:
I would just do:

```cmake
add_swift_target_library(swiftCompatibility50 TARGET_LIBRARY STATIC
  ProtocolConformance.cpp
  Overrides.cpp
  TARGET_SDKS ${SWIFT_APPLE_PLATFORMS}
  C_COMPILE_FLAGS ${swift_runtime_library_compile_flags}
  LINK_FLAGS ${swift_runtime_linker_flags}
  INSTALL_IN_COMPONENT stdlib)
```